experiments/benchmark_result.py

```diff
@@ -0,0 +1,335 @@
+
+        def loss_tracin(params, data_target_pair):
+            image, label = data_target_pair
+            image_t = image.unsqueeze(0)
+            label_t = label.unsqueeze(0)
+            loss = nn.CrossEntropyLoss()
+            yhat = torch.func.functional_call(model_details["model"], params, image_t)
+            return loss(yhat, label_t.long())
+
+        task = AttributionTask(
+            model=model_details["model"].to(args.device),
+            loss_func=loss_tracin,
+            checkpoints=model_details["models_full"][0:10]
+            if args.method == "TracIn"
+            else model_details["models_full"][0],
+        )
+
+    if args.method in ["TRAK-1", "TRAK-10", "TRAK-50"]:
+
+        def loss_trak(params, data_target_pair):
+            image, label = data_target_pair
+            image_t = image.unsqueeze(0)
+            label_t = label.unsqueeze(0)
+            loss = nn.CrossEntropyLoss()
+            yhat = torch.func.functional_call(model_details["model"], params, image_t)
+            logp = -loss(yhat, label_t)
+            return logp - torch.log(1 - torch.exp(logp))
+
+        task = AttributionTask(
+            model=model_details["model"].to(args.device),
+            loss_func=loss_trak,
+            checkpoints=model_details["models_half"][
+                0 : int(args.method.split("-")[1])
+            ],
+        )
+
+        def m_trak(params, image_label_pair):
+            image, label = image_label_pair
+            image_t = image.unsqueeze(0)
+            label_t = label.unsqueeze(0)
+            loss = nn.CrossEntropyLoss()
+            yhat = torch.func.functional_call(model_details["model"], params, image_t)
+            p = torch.exp(-loss(yhat, label_t.long()))
+            return p
+
+    if args.method == "RPS":
+
+        def loss_rps(pre_activation_list, label_list):
+            loss_fn = nn.CrossEntropyLoss()
+            return loss_fn(pre_activation_list, label_list)
+
+        model = model_details["model"].to(args.device)
+        model.load_state_dict(torch.load(model_details["models_full"][0]))
+
+        task = AttributionTask(
+            model=model_details["model"].to(args.device),
+            loss_func=loss_rps,
+            checkpoints=model_details["models_full"][0],
+        )
+
+    # 3. start running the TDA methods
+    best_result = 0
+    best_config = None
+
+    if args.method in ["if-explicit", "if-cg", "if-lissa", "if-arnoldi"]:
+        for ihvp_config in IHVP_SEARCH_SPACE[args.method]:
+            print(ihvp_config)
+            attributor = ATTRIBUTOR_DICT[args.method](
+                task=task,
+                device=args.device,
+                **ihvp_config,
+            )
+            attributor.cache(train_loader_cache)
+            with torch.no_grad():
+                score = attributor.attribute(train_loader, test_loader)
+
+            # compute metrics
+            if args.metric == "lds":
+                score = -score
```

Comment:
why?

Reply:
I think it's better to make the API of all metrics consistent

Reply:
Oh, this should be the LDS groundtruth calculation problem, since the LDS groundtruth is calculated by `loss` for now, it should be `-loss` since LDS using L_{+z} and LOO using L_{-z} (where z is some training sample).

I guess the best solution is to make all the LDS groundtruth we upload to be negative of the current version?

Reply:
> Or raise and merge a PR about the API change first and then update this PR?
> 
> We may also address issues #26 and #29 first.

Thanks! If we decided to change the groundtruth to make this API consistent, we may fix these two issues together in a seperated PR.

Reply:
Fixed the groundtruth.